media/js/base/protocol/protocol-menu.js

```diff
@@ -0,0 +1,341 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Copied from Protocol, to be backported along with nav updates.
```

Comment:
Yeah I'm not happy with the duplication. The only significant changes are the names, since I removed the `mzp-` prefix to avoid clashing with Protocol. The plan is that once I can backport the redesign into Protocol these duplicates can be removed, hence the comment.

BUT we could also keep the `mzp-` prefixes in the updated nav and the Protocol JS should all still work without duplicating it. But then we'll need to be careful that there aren't other name collisions or that we don't make a bigger mess. I also thought this might simplify a future update when we get everything into Protocol. It seemed like duplicating was the lesser crime but I can be convinced otherwise.

Reply:
I think as long as we're not comitted to maintaining this code in bedrock long term (vs in protocol) then that's fine. Mostyly I just wasn't sure how much of this needs to be re-reviewed, but if it's just changin selectors then I think that's fine 👍